### PR TITLE
Pin the component.yaml version in all the samples.

### PR DESCRIPTION
**Pull request by hongye-sun**
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/pipelines/1037)

### CODE REVIEW

File: components/gcp/dataproc/submit_hadoop_job/README.md [hunk 1]
```diff
@@ -72,7 +72,7 @@ To continue to run the sample, make sure that the service account of the noteboo
 !gsutil rm $OUTPUT_GCS_PATH/**
 ```
 
-    CommandException: No URLs matched: gs://hongyes-ml-tests/dataproc/hadoop/output/**
+    CommandException: No URLs matched: gs://hongyes-ml-tests/dataproc/hadoop/output/**
```
Comment (gaoning777) [inline components/gcp/dataproc/submit_hadoop_job/README.md]: Nit: Could we change the bucket name from hongyes-ml-tests to ml-pipeline-tests or something?

Comment (Ark-kun) [reply components/gcp/dataproc/submit_hadoop_job/README.md]: +1

Comment (hongye-sun) [reply components/gcp/dataproc/submit_hadoop_job/README.md]: Those output cells are mistakenly added. I have removed them.

**Pull request merged**